.github/workflows/ferretdb_go_tests.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Go
+on:
+  pull_request:
+    types:
+      - unlabeled # if GitHub Actions stuck, add and remove "not ready" label to force rebuild
+      - opened
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - main
+      - ferretdb
+  schedule:
+    - cron: "12 0 * * *"
+
+env:
+  GOPATH: /home/runner/go
+  GOCACHE: /home/runner/go/cache
+  GOLANGCI_LINT_CACHE: /home/runner/go/cache/lint
+  GOMODCACHE: /home/runner/go/mod
+  GOPROXY: https://proxy.golang.org
+  GOTOOLCHAIN: local
+
+jobs:
+  test:
+    name: Test
+
+    # CI cannot find ubicloud-standard-4 runner and displays the following message:
+    # `Waiting for a runner to pick up this job...`
```

Comment:
I couldn't get `runs-on: ubicloud-standard-4` to start job, it was waiting.

Reply:
That should be resolved now, let's use Ubicloud